plugin/trino-hive/src/test/java/io/trino/plugin/hive/coercions/TestIntegerNumberToDoubleCoercerParquet.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive.coercions;
+
+import io.trino.plugin.hive.parquet.ParquetTypeTranslator;
+import io.trino.spi.block.Block;
+import io.trino.spi.type.BigintType;
+import io.trino.spi.type.IntegerType;
+import io.trino.spi.type.SmallintType;
+import io.trino.spi.type.TinyintType;
+import io.trino.spi.type.Type;
+import org.apache.parquet.schema.PrimitiveType;
+import org.apache.parquet.schema.Types;
+
+import static io.trino.spi.predicate.Utils.nativeValueToBlock;
+import static io.trino.spi.type.DoubleType.DOUBLE;
+import static org.apache.parquet.schema.LogicalTypeAnnotation.IntLogicalTypeAnnotation.intType;
+import static org.apache.parquet.schema.PrimitiveType.PrimitiveTypeName.INT32;
+import static org.apache.parquet.schema.PrimitiveType.PrimitiveTypeName.INT64;
+
+public class TestIntegerNumberToDoubleCoercerParquet
```

Comment:
I don't see the point of running this test separately for parquet. The coercion logic is mostly outside of parquet, the only part in parquet is the selection of TypeCoercer based on parquet and Trino types and the page source factory change. We should just have a query runner test which creates parquet file with tinyint/smallint/int/bigint types and then reads that back as a table with all double types. If that's redundant with the product tests we already have, then we can skip it